src/org/labkey/test/tests/KnitrReportTest.java

```diff
@@ -199,13 +199,11 @@ private void verifyAdhocReportDependencies(String viewName, String dependencies)
 
         createKnitrReport(rmdDependenciesReport, RReportHelper.ReportOption.knitrMarkdown);
 
-        pauseJsErrorChecker(); // Don't fail due to "$ is not a function"
-        {
-            _rReportHelper.clickReportTab();
-            waitForElement(Locator.id("mtcars_table"));
-            assertElementNotPresent(Locator.id("mtcars_table_wrapper")); // Created by jQuery
-        }
-        resumeJsErrorChecker();
+        int errorCountBefore = getServerErrorCount();
+        _rReportHelper.clickReportTab();
+        waitForElement(Locator.id("mtcars_table"));
+        assertElementNotPresent(Locator.id("mtcars_table_wrapper")); // Created by jQuery
+        checkExpectedErrors(errorCountBefore + 1); // JavaScript error: "$(...).dataTable is not a function"
```

Comment:
`checkExpectedErrors(1)` is preferable. We want exactly one error.
We should also make sure that it is the error we expect.
```suggestion
        _rReportHelper.clickReportTab();
        waitForElement(Locator.id("mtcars_table"));
        assertElementNotPresent(Locator.id("mtcars_table_wrapper")); // Created by jQuery
        Assertions.assertThat(getServerErrors()).as("Server errors").contains("$(...).dataTable is not a function");
        checkExpectedErrors(1); // JavaScript error: "$(...).dataTable is not a function"
```